docs/docs_source/wopi/files/RefreshLock.rst

```diff
@@ -14,17 +14,35 @@ RefreshLock
     The refreshed lock must expire automatically after 30 minutes unless it is modified by a subsequent WOPI
     operation, such as :ref:`Unlock` or :ref:`RefreshLock`.
 
+    ..  include:: /fragments/priorlock.rst
+
+    ..  include:: /fragments/lock409.rst
+
+    ..  include:: /fragments/no_lock_id.rst
+
+    ..  note::
+        Since there is no GetLock operation defined in WOPI, Office Online will sometimes make a RefreshLock call to
```

Comment:
@MarcinJuraszek I looked at the WopiStorage code and it looks like the only one that actually cares about the OldLockValue is the Lock request. I might have missed it though.